content/releases/3-0-1.md

```diff
@@ -0,0 +1,124 @@
+It also clears the way for the new, improved Work Calendar service (still in release candidacy).
+
+_Release date:_
+23 Jul 2024
+
+{{< notice "warning" >}}
+This release includes a few minor breaking changes, review the [upgrade](#upgrade) instructions for details.
+{{< /notice >}}
+
+### BPMN
+
+**Features**
+
+- Error boundary events now support call-activity tasks. If a task in the called process aborts, the error boundary captures that event.
+- Support Custom [Certificate authorities](https://en.wikipedia.org/wiki/Certificate_authority) so that the BPMN service can run in networks with internal certificates.
+- [Message events](http://localhost:1313/how-to/bpmn/bpmn-elements/#events) now support topics that use the `+` wildcard.
+- Better support for [Work calendars]({{< relref "/how-to/work-calendars" >}})
+  - When configured, calendar data persists to a timeseries DB.
+  - `Breaking` Work calendars now query for `WorkCalendarDefinitions` not `workCalendars`.
+  This makes it possible to configure calendars in the UI.
+  - Work calendar execution is now based on Hierarchy Scope.
+
+**Fixes**
+- Reduce number of retries in [BPMN recovery]({{< relref "/deploy/maintain/bpmn-nodes/" >}}), speeding up response times when node does not exist.
+ For the user's experience, the delay was particularly notable on [API BPMN triggers]({{< relref "/how-to/bpmn/trigger-workflows/#start-a-workflow-from-an-api" >}}) when the specified workflow did not exist.
+- Capture panics in JSONata go for better recovery and discovery.
+- `Breaking`. Limit BPMN users to only tokens with specific audience and scope.
+- Fall back to looking up specs in the database. Fixes issue when spinning up new development containers.
+- Include metadata fields for `jobResponse`. Fixes panic if node tried to access these fields.
+- Respect [`debug`](/how-to/bpmn/debug-workflows/#adding-the-debug-flag) flag for BPMNs triggered with `createAndRunBpmn`.
+- Reuse expected Audience for clients to match documented client name.
+- Do not create calendar entry for end events.
+- Allow `__traceDebug` to be set from [variable context]({{< relref "/how-to/bpmn/create-workflow/#access-process-variable-context" >}}).
+
+
+### Schema
+
+**Features**
+- `breaking`. Add `id` and `label` to `stateTransitionInstance`.
+- Add menu URL for portal in UI (preparation for future work in UI).
+- Add filter to query material actual by a material requirement and vice-versa.
+- Link `workCalendarEntries` to equipment versions.
+- Include Kafka in the enum values for data sources.
+- Added optional `InformationObjectTypeEnum` to `OperationsEventRecordEntry`.
+
+**Fixes**
+- Add 0-n relationship from `materialRequirements` to `materialActual`.
+- The `segmentResponse` entity can now have multiple `jobResponses` attached.
+- `Breaking`. Change `stateModelTransition` from `state` to a relationship.
+- link `operationsEventDefinitionProperty` to `operationsEventProperty`.
```

Comment:
Fix inverse relationship between `operationseventDefinitionProperty` and `operationsEventProperty`